source/features/bypass-checks.tsx

```diff
@@ -1,19 +1,20 @@
+import mem from 'mem';
 import onetime from 'onetime';
 import {observe} from 'selector-observer';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
-import fetchDom from '../helpers/fetch-dom';
+import * as api from '../github-helpers/api';
+import {getRepoURL} from '../github-helpers';
 
-async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
-	const directLink = await fetchDom<HTMLAnchorElement>(
-		detailsLink.href,
-		'[data-hydro-click*="check_suite.external_click"]'
-	);
+const getDirectLink = mem(async (runId: number): Promise<string> => {
```

Comment:
The API is already memoize, no need to do it again

Reply:
https://github.com/sindresorhus/refined-github/blob/022e418940b98f7dfa7f4e2c532b2b83782ca45a/source/features/deep-reblame.tsx#L16

I keep doing it since I remember from here

Reply:
It can be removed from there too. It would only make sense if it saved heavy operations (e.g. HTTP requests), but what follows is just a few conditions.